mousestyles/density_estimation.py

```diff
@@ -0,0 +1,162 @@
+# coding: utf-8
+
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+from mousestyles.data import load_movement
```

Comment:
For style, I think we load numpy and pandas first, then load our own package after